src/services/plan-service.test.js

```diff
@@ -351,7 +351,7 @@ describe('Doppler plan client', () => {
     const plans = await planService.getPlans(currentPlan, 'standard', 'subscribers', planList);
 
     // Assert
-    expect(plans.length).toBe(2);
+    expect(plans.length).toBe(0);
```

Comment:
I think this test is no longer valid or you could change the name. 
Instead of 
`it('should get higher subscriber plans when user type: monthly-deliveries', async () => {`
`it('should get no subscriber plan when user type: monthly-deliveries', async () => {`
What do you think? remove it or change the name? 🤔

Reply:
Maybe it could be better to rename the test, so we can be sure that subscriber plan is not going to be showed when plan user type is monthly deliveries 😉